### ISSUE TITLE
Runs with an out of bounds error

### ISSUE THREAD

**Issue by JacksonChen666** [cyqsimon/ntfysh-spec#3]
Upstream ntfy RPM works fine, this COPR package just doesn't work at all.

Logs:
```
root@laptop-server:~# /usr/bin/ntfy
panic: runtime error: slice bounds out of range [:7] with length 0

goroutine 1 [running]:
main.main()
	/builddir/build/BUILD/ntfy-2.7.0/main.go:27 +0x385
```

Version: 2.7.0-2.fc39

**Comment by cyqsimon** [comment]
Yikes. I know exactly what's wrong. The code to automatically fetch the commit hash must have failed. I'll fix it ASAP.

**Comment by cyqsimon** [comment]
Should be fixed in 2.7.0-3.

**Comment by JacksonChen666** [comment]
> Should be fixed in 2.7.0-3.

yep, fixed
